test/Scripts.Integration.Test/SentrySetup.cs

```diff
@@ -0,0 +1,89 @@
+using UnityEditor;
+using UnityEditor.PackageManager.Requests;
+using UnityEditor.PackageManager;
+using UnityEngine;
+using System.Linq;
+
+[InitializeOnLoad]
+public static class Startup
+{
+    static AddRequest AddRequest;
+    static ListRequest ListRequest;
+
+    const string SentryPackageName = "io.sentry.unity";
+    const string SentryUPMUrl = "https://github.com/getsentry/unity.git";
+
+    static void LogDebug(string message)
+        => Debug.LogFormat(LogType.Log, LogOption.NoStacktrace, null, $"Sentry setup: {message}");
+
+    static void LogError(string message)
+        => Debug.LogFormat(LogType.Error, LogOption.NoStacktrace, null, $"Sentry setup: {message}");
+
+    static void ExitIfBatchMode(int code)
+    {
+        if (Application.isBatchMode)
+        {
+            EditorApplication.Exit(code);
+        }
+    }
+    static Startup()
+    {
+        LogDebug("checking if Sentry is installed");
+        CheckIfSentryIsInstalled();
+    }
+
+    static void RequestSentryInstall()
+    {
+        AddRequest = Client.Add(SentryUPMUrl);
+        EditorApplication.update += SentrySetupProgress;
+    }
+
+    static void CheckIfSentryIsInstalled()
+    {
+        ListRequest = Client.List();
+        EditorApplication.update += SentryIsInstalledProgress;
+    }
+
+    static void SentrySetupProgress()
+    {
+        if (AddRequest.IsCompleted)
+        {
+            EditorApplication.update -= SentrySetupProgress;
+
+            if (AddRequest.Status == StatusCode.Success)
+            {
+                LogDebug("SUCCESS");
+                ExitIfBatchMode(0);
+            }
+            else if (AddRequest.Status >= StatusCode.Failure)
+            {
+                LogError("FAILED");
+                LogError(AddRequest.Error.message);
+                ExitIfBatchMode(-1);
+            }
+        }
+    }
+
+    static void SentryIsInstalledProgress()
+    {
+        if (ListRequest.IsCompleted)
+        {
+            EditorApplication.update -= SentryIsInstalledProgress;
+
+            if (ListRequest.Status >= StatusCode.Failure)
+            {
+                LogError(ListRequest.Error.message);
+            }
+            else if (ListRequest.Result.Any(p => p.name.Contains(SentryPackageName)) == false)
```

Comment:
`io.sentry.unity` would match for `io.sentry.unity-dev` if using Contains.

Can we match exactly here?

```suggestion
            else if (ListRequest.Result.Any(p => p.name == SentryPackageName) == false)
```